models/topics.model.js

```diff
@@ -24,3 +24,13 @@ exports.selectAllArticles = () => {
         return rows;
     })
 }
+
+exports.selectComments = (article_id) => {
+  const query = 'SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at desc;'
+  return db.query(query, [article_id]).then(({rows}) => {
+    if (!rows.length) {
+      return Promise.reject({ status: 404, msg: "Not found" });
+    }
+    return rows
+  })
```

Comment:
We should be checking why we are getting an empty array here 👀 is it because the article exists but has no comments or is it because the article doesn't exist? You be will seeing the same result of an empty array for two very different reasons. Think about how you could check the articles table first to see if the ID exists 🤔 would you want to see an error first and if not then you could query the comments table 💭